### PR TITLE
Dev cognitiveservices form recognizer v2.1 preview.2

### PR DESCRIPTION
**Pull request by xingong202**
### What's the change

This change updates CognitiveServices/FormRecognizer layout API to add "boundingBox" property to "DataTable" object.

### Why make this change

1. API consistency.
All other entities in layout API's output, including text lines, words, selectionMarks, even tableCell have "boundingBox" property, except for table itself.
2. Ease of use.
There are feedback after rolling out of layout API, both internally and externally, that the bounding box of table will be a welcome addition to the API. Without it, users have to calculate convex hull of all enclosed table cells, that is not only inaccurate, but also difficult for developer.

### How the change is made
1. API verison bump
This new property will start to appear in 2.1-preview.2. OpenAPI Hub was used to copy existing API definition and examples, update readme.md, and change API version string in all related files.
2. Adding new "boundingBox" property
This is a simple change to add one more property "boundingBox" into existing "DataTable" definition. The "boundingBox" shares the same definition as any other "boundingBox" in layout API.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json [hunk 1]
```diff
@@ -0,0 +1,1972 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.1-preview.1",
```
Comment (ChrisHMSFT) [inline specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json:4]: Should this be 2.1-preview.2 based on the title?

Comment (xingong202) [reply specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json]: Yes, it should. Thanks!

File: specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/examples/AnalyzeBatch.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "body": {}
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "{endpoint}/formrecognizer/v2.1-preview.1/custom/models/{modelId}/analyzeResults/{resultId}"
```
Comment (bojunehsu) [inline specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/examples/AnalyzeBatch.json]: Please update version across all examples.

**Review by bojunehsu — CHANGES_REQUESTED** [review]
We will most likely need to merge this PR with https://github.com/Azure/azure-rest-api-specs/pull/10805.

@ramparab

**Comment by maririos** [comment]
@xingong202 is the idea to merge this change as part of https://github.com/Azure/azure-rest-api-specs/pull/11088 or manage it in separate PRs?
Also, do you have an ETA on when the change is going to be merged (either here or in the other PR)?

File: specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/examples/AnalyzeBatch.json (hunk 2 above)
Comment (xingong202) [reply specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/examples/AnalyzeBatch.json]: The version is already updated. The PR contains multiple commits, some of them are auto generated. The initial commit (created by a tool) simply copy everything from preview.1, the second commit (also created by the tool) updated all version numbers to preview.2

**Comment by anuchandy** [comment]
Closing it due to inactivity, feel free to reopen (after rebase with master) or a new PR with the changes.

**Pull request closed without merging**